content/docs/alerting/configuration.md

```diff
@@ -442,6 +442,15 @@ links:
 
 # The HTTP client's configuration.
 [ http_config: <http_config> | default = global.http_config ]
+
+# The part or component of the affected system that is broken.
+[ component: <tmpl_string> ]
+
+# A cluster or grouping of sources.
+[ group: <tmpl_string> ]
+
+# The class/type of the event.
+[ class: <tmpl_string> ]
```

Comment:
lets move it before the `http_config` entry to be consistent with the other sections.

Reply:
I've done that now. There's another problem though, this file was deleted from the repo in #1657 and doesn't seem to exist anywhere else. Per that change, it looks like these AlertManager docs might be moving to the alertmanager repo, and pulled in here during build. I can't seem to find it there either though. 

@roidelapluie any thoughts or suggestions?

Reply:
@grdryn indeed you need to move your changes here:
https://github.com/prometheus/alertmanager/blob/master/docs/configuration.md
sorry for the annoyance